streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -892,8 +892,8 @@ void process(final ProcessorSupplier<? super K, ? super V> processorSupplier,
      * <p>
      * Because a new key is selected, an internal repartitioning topic may need to be created in Kafka if a
      * later operator depends on the newly selected key.
-     * This topic will be as "${applicationId}-XXX-repartition", where "applicationId" is user-specified in
-     * {@link  StreamsConfig} via parameter {@link StreamsConfig#APPLICATION_ID_CONFIG APPLICATION_ID_CONFIG}, "XXX" is
+     * This topic will be as "${applicationId}-&lt;name&gt-repartition", where "applicationId" is user-specified in
```

Comment:
We used `${..}` to signify parameter substitution for applicationId. Should we do the same for name?

Reply:
Maybe -- however, application.id is a configuration parameter and cannot be specified on the method while name is a local parameter (my idea was to use different markup to distinguish both)

Reply:
I'm inclined to go with @mjsax as it's a local parameter.  @vvcephei if you insist I can change it.

Reply:
Ok, that's fine with me!